debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/RecordMakersTest.java

```diff
@@ -76,7 +77,7 @@ public void shouldGenerateRecordForInsertEvent() throws InterruptedException {
         Struct key = (Struct) record.key();
         Struct value = (Struct) record.value();
         assertThat(key.schema()).isSameAs(record.keySchema());
-        assertThat(key.get("_id")).isEqualTo(objId.toString());
+        assertThat(key.get("id")).isEqualTo(JSONSerializers.getStrict().serialize(objId));
```

Comment:
I did some manual testing and noticed that the key `1` (i.e. of type integer) is serialized into `"1.0"`. Is this expected? One could only read this back as a floating point number it seems?

Reply:
Ah, that seems to be caused by the Mongo shell client (which I used for testing):

> The mongo shell treats all numbers as floating-point values by default

I get "1" when inserting it as `NumberInt(1)`.

Reply:
I would have been surprised because I'm pretty sure that I had done some serialization unit tests for simple id types in a different context some time ago which seemed to work :)